### ISSUE TITLE
Search by tags on windows 10

### ISSUE THREAD

**Issue by GitManu84** [Dijji/FileMeta#95]
Hi :)
I have enabled tags for .exe files to search for applications installed on Windows 10 by dividing them by categories but the "tag:" syntax does not work in the taskbar search.
Is there a way to enable the search for .exe files via tags also in the taskbar search?

Thanks.

**Comment by Dijji** [comment]
It's all in the name. You need keywords:=tagval
It is because the name of the property is System.Keywords. Tag is just the display name for the property, which is shown in most places, but does not apply to the search. Gotcha!

Dijji

**Comment by Dijji** [comment]
Of course, you also need to use Indexing Options to make sure that the folders containing your executables are indexed. By default in Windows 10,  .exe file properties are indexed, so that part should be fine.

**Comment by GitManu84** [comment]
Hi, folders containing exe files are indexed. I tried keywords:=tagval but it doesn't work.

**Comment by Dijji** [comment]
I see what you're trying to do now. What makes you think that tag: might be one of the set of supported filters? I cannot discover for the moment any hint that the support goes beyond the filters that can be set up using the UI that pops up above the search area  to.

**Comment by GitManu84** [comment]
Nothing I just tried =D, I thought it worked because with other types of files the syntax "tag:" it works (example for .jpeg).
In the explorer search box the syntax "tag:" it works perfectly.

**Comment by Dijji** [comment]
Well I am learning something here. My understanding of the search box syntax was seriously dated: it seems they have simplified and improved it a good deal.  As you say, tag: works perfectly, including picking up tagged .exe files. 
But none of the syntax seems to work in the taskbar. For instance, "ext: txt" comes up with nothing useful, even though the taskbar search is apparently based on the same indexing engine, according to the documentation. It seems that this is just how things are, for the moment at least.

**Comment by GitManu84** [comment]
We hope that Windows implements these syntax also in the taskbar, it would be really useful instead of doing the search by opening a window of the explorer every time.
Thanks for the info, file meta association is a great utility.
